doc/release/1.15.0-notes.rst

```diff
@@ -114,6 +117,12 @@ This only enables you to compile NumPy for one specific platform at a time.
 Creating a full iOS-compatible NumPy package requires building for the 5
 architectures supported by iOS (i386, x86_64, armv7, armv7s and arm64), and
 combining these 5 compiled builds products into a single "fat" binary.
+``np.quantile`` and ``np.nanquantile``
```

Comment:
nit: missing newline due to the merge / rebase

Reply:
Addressed.